src/sentry/notifications/notifications/rules.py

```diff
@@ -39,6 +39,7 @@
 from sentry.services.hybrid_cloud.user_option.service import get_option_from_list
 from sentry.types.actor import Actor
 from sentry.types.group import GroupSubStatus
+from sentry.types.integrations import ExternalProviderEnum, ExternalProviders
```

Comment:
```suggestion
from sentry.integrations.types import ExternalProviderEnum, ExternalProviders
```
i moved the imports as part of the new messaging refactor, will delete the old module so folks don't get confused.